### DISCUSSION TITLE
4.0.0

### DISCUSSION THREAD

**Discussion by dpfaffenbauer** [coreshop/CoreShop#2486] [Announcements]
# 4.0.0

This is the final Release of CoreShop 4.0.0 for Pimcore 11 🎉 

This release is the same as 3.2.0 compatible with Pimcore 11.

**Comment by averatec1337** [comment]
@dpfaffenbauer 
![image](https://github.com/coreshop/CoreShop/assets/2456974/cd1fe827-29b5-4aae-bc84-893f8302238e)
is it intentional to not set alias for custom controllers?

**Reply by dpfaffenbauer** [reply]
We got rid of all aliases in CoreShop 4 and use FQCN now.

**Reply by averatec1337** [reply]
I've just found this in another discussion, thank you for quick response.
